packages/aws-cdk/package.json

```diff
@@ -103,5 +99,23 @@
   "engines": {
     "node": ">= 10.3.0"
   },
-  "stability": "stable"
+  "stability": "stable",
```

Comment:
Why do we need this duplicated configuration everywhere? For example the reporter types should be the same for all modules!